src/components/badge-list/index.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames'
+import Badge from 'components/badge'
+import { BadgeProps } from 'components/badge/types'
+import Tooltip from 'components/tooltip'
+import s from './badge-list.module.css'
+
+export interface Badge {
+	text: string
+	icon?: React.ReactElement
+	tooltip?: string
+}
+
+interface BadgeListProps {
+	badges: Array<Badge>
+	className?: string
+	size?: BadgeProps['size']
+	type?: BadgeProps['type']
+	color?: BadgeProps['color']
+}
+
+export default function BadgeList({
+	badges,
+	className,
+	size = 'medium',
+	color = 'neutral',
+	type = 'filled',
+}: BadgeListProps) {
+	return (
+		<ul className={classNames(s.badgeList, className)}>
+			{badges.map((badge: Badge) => {
+				return (
+					<li key={badge.text} className={s.badge}>
+						{badge.tooltip ? (
+							<Tooltip label={badge.tooltip}>
+								<Badge
+									icon={badge.icon}
+									text={badge.text}
+									color={color}
+									type={type}
+									size={size}
```

Comment:
Might want to defer to the badge data here if it is passed, since these props are optional

Reply:
Not 100% quite sure I follow!

Adding context that might help, if I assume what you are suggesting correctly:

I am passing in the color / type / size as passed at the `badge-list` level here (the individual badge interfaces I have is a reduced interface on the actual Badge component, so you cannot set the color / type / size of the badge directly).

Reply:
In the interface, the props are defined as optional so they could be `undefined` and then you would be using the badge default anyways. I think we should do something like this so that one could drive with the badge data as an option. 

```suggestion
size={badge.size || size}
```

Reply:
Ah! I see!  That makes it more clear thank you — yeah this should be added!

Reply:
Oh!  I handle this in the destructuring; so this will never be undefined:

https://github.com/hashicorp/dev-portal/pull/1687/files#diff-ddce842efc461cbfc09ed3f97cd1630552d470994031f2425f72c1387e617781R24-R25